source/server/http/admin.cc

```diff
@@ -316,7 +316,9 @@ Http::Code AdminImpl::handlerStats(const std::string& url, Buffer::Instance& res
 std::string AdminImpl::formatTagsForPrometheus(const std::vector<Stats::Tag>& tags) {
   std::vector<std::string> buf;
   for (const Stats::Tag& tag : tags) {
-    buf.push_back(fmt::format("{}={}", tag.name_, tag.value_));
+    const std::string label_value = StringUtil::replace(tag.value_, ".", "_");
```

Comment:
You can just use std::replace here. You don't need to add a new function.

Reply:
Ok cool.

Reply:
Ok cool.